lib/core/src/persist/backup.rs

```diff
@@ -33,6 +41,92 @@ impl Persister {
 
         Ok(())
     }
+
+    #[cfg(all(target_family = "wasm", target_os = "unknown"))]
+    pub async fn backup_to_indexed_db(&self, db_name: &str) -> Result<()> {
```

Comment:
Can we move also the backup_to_indexed_db & restore_from_index_db to the wasm crate?

Reply:
Done in [c8bec8f](https://github.com/breez/breez-sdk-liquid/pull/851/commits/c8bec8fd64ce12ee8e9530d67c1234a955b35e8b)